src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -61,7 +61,10 @@ export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanProperti
 			this._primaryTarget = localize('executionPlanPropertiesEdgeOperationName', "Edge");
 		}
 
-		let topTitleText = localize('executionPlanComparisonPropertiesTopOperation', "Top operation: {0}", this._primaryTarget);
+		const topTitleText = this._orientation === ExecutionPlanCompareOrientation.Horizontal
```

Comment:
`primaryTitleText` might be more appropriate given the orientation differences, and I'd update this function to be `setPrimaryElement`. In fact there appears to be a number of places that still refer to "top" - all of which I'd suggest updating to primary (and bottom to secondary).